app/.env

```diff
@@ -0,0 +1,2 @@
+SECRET_KEY=sk_test_0hQ5W9JaWZEkDLd6CjH0wl4a
+PUBLISHABLE_KEY=pk_test_oV4RE2n0vxcKmOVfiTpdaFUx
```

Comment:
We should add `app/.env` file into the `.gitignore` file because we definitely don't want to track our API keys for third-party services.